sound-bubble/imports/ui/components/Home/SongLog.jsx

```diff
@@ -1,23 +1,18 @@
 import React, { Component } from 'react';
 import Song from '../Song';
+import compose from 'recompose/compose';
 import { connect } from 'react-redux';
+import { withTracker } from 'meteor/react-meteor-data';
 import { fetchGroupSongLogs } from '../../actions/homeActions';
-import Songs from '../../../api/songs';
 import GroupButton from './GroupButton';
+import UserSongs from '../../../api/userSongs';
 
 class SongLog extends Component {
   getSongDetails() {
-    return this.props.groupRecentTracks
-      .filter(t => t.show)
-      .map(t => {
-        return (
-          <Song
-            key={t.songId + t.userId + t.timestamps}
-            track={t}
-            home={true}
-          />
-        );
-      });
+    console.log('groupRecent', this.props.groupRecentTracks);
```

Comment:
still need?

Reply:
I kinda want to keep this as song logs are buggy sometimes. Yancey and I have seen users in one group get the same, duplicated song logs from one user (happened in our last demo, too).

I think it's something to do with the tokens but I don't understand how that happened. I would want to keep this console log until groups and the load more button are fully implemented.